### PR TITLE
App development

### PR DESCRIPTION
**Pull request by DiamondNath07**
### Essence of the PR
This PR attempts to re-create this application considering the ES6 syntax and still keeping all the functionalities of the applications which are to create, read, update and delete books from the book library.

### This PR accomplishes the following
- Display the current date and time using the Luxon library
- Sectionalise codes into smaller chunks called modules.
- Appropriate use of the Import command in the main entry point
- Appropriate use of const and let
- Effective use of local storage to save books

### CODE REVIEW

File: modules/BookDisplay.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { saveDataToLocalStorage } from './LocalStorage.js';
+
+export default function showBooks(books) {
+  function addElement(elementType, parent, className) {
+    const element = document.createElement(elementType);
+    element.classList.add(className);
+    parent.appendChild(element);
+    return element;
+  }
+
```
Comment (Munsa1) [inline modules/BookDisplay.js]: - [ ] This milestone encourages us to use E6 JavaScript coding standards. Kindly use an arrow function here

File: modules/LocalStorage.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const storageKey = 'awesomeBooks';
+function loadDataFromLocalStorage() {
+  const dataLoaded = JSON.parse(localStorage.getItem(storageKey));
+  if (dataLoaded == null) {
+    return [];
+  }
```
Comment (Munsa1) [inline modules/LocalStorage.js]: - [ ] Kindly use arrow format for functions here, as this is the given requirement

File: modules/LocalStorage.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  return dataLoaded;
+}
+
+function saveDataToLocalStorage(arrBooks) {
+  localStorage.setItem(storageKey, JSON.stringify(arrBooks));
+}
+
```
Comment (Munsa1) [inline modules/LocalStorage.js]: - [ ]Again here kindly use arrow format for functions here, as this is the given requirement

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @DiamondNath07, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._
- [ ] Currently it is possible to add a book without an author, kindly fix this bug. Take a look at the screenshot below and look at the second entry
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @DiamondNath07 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**